src/charts/charts.less

```diff
@@ -11,6 +11,32 @@ pf-c3-chart {
     width: 100%;
   }
 }
+.pct-donut-chart-pf {
+
+  .pct-donut-chart-pf-block {
+    .pct-donut-chart-pf-label {
+      display: block;
+    }
+  }
+
+  .pct-donut-chart-pf-left,
+  .pct-donut-chart-pf-right {
+
+    pf-c3-chart {
+      display: inline-block;
+    }
+
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .pct-donut-chart-pf-left {
+    flex-direction: row-reverse;
+  }
+}
+
```

Comment:
@jeff-phillips-18 @dtaylor113  the less has been moved over to core Patternfly PR  720